contrib/storage-hive/core/src/test/java/org/apache/drill/exec/impersonation/hive/TestSqlStdBasedAuthorization.java

```diff
@@ -78,6 +80,7 @@
 
   @BeforeClass
   public static void setup() throws Exception {
+    assumeTrue("Skipping tests since Hive supports only JDK 8.", HiveTestUtilities.supportedJavaVersion());
```

Comment:
Can't we move check to `BaseTestHiveImpersonation` and extend from some common Hive test class?